api-client/src/models/CityModel.ts

```diff
@@ -32,13 +27,8 @@ class CityModel {
     prefix: string | null | undefined
     latitude: number | null
     longitude: number | null
-    aliases: Record<
-      string,
-      {
-        longitude: number
-        latitude: number
-      }
-    > | null
+    aliases: Record<string, { longitude: number; latitude: number }> | null
+    boundingBox: { latitude: number; longitude: number }[] | null
```

Comment:
Could you extract the bounding box type here? You seem to reuse `{ latitude: number; longitude: number }` quite often

Reply:
Another question is if we should not use the standard format for a bounding box refer to geoJSON
```js
/**
 * Bounding box
 * https://tools.ietf.org/html/rfc7946#section-5
 */
export type BBox = [number, number, number, number] | [number, number, number, number, number, number];
```

Reply:
@f1sh1918 sorry I don't have time to adjust this right now, but feel free to change this :)